src/main/java/lotto/domain/WinnerCount.java

```diff
@@ -0,0 +1,39 @@
+package lotto.domain;
+
+import java.math.BigDecimal;
+import java.util.HashMap;
+import java.util.Map;
+
+public class WinnerCount {
+    public static final int FORWARD_WINNER = 2;
+    private Map<Integer, Integer> winnerCountMap;
```

Comment:
```suggestion
    private Map<CountPrice, Integer> winnerCountMap;
```

객체를 key로 가져도 좋을것? 같기도해서 관점 공유해봅니다 :)

Reply:
제대로 한건지는 모르겠으나, 반영했습니다!